.github/pull_request_template.md

```diff
@@ -6,7 +6,7 @@ _Summarize your changes here : explain what, how, and why. Be as explicict as yo
 ## Checklist
 
 - [ ] My PR is related to an opened issue : #
-- [ ] I've read the [contributing guidelines](../CONTRIBUTING.md)
+- [ ] I've read the [contributing guidelines](https://github.com/CleverCloud/documentation/blob/main/CONTRIBUTING.md)
```

Comment:
```suggestion
- [ ] I've read the [contributing guidelines](/blob/main/CONTRIBUTING.md)
```

Should be enough ?

Reply:
@davlgd Unless github rewrites the URL I'm not sure /blob/main/ will work, trying it by changing it to that using dev tools on this PR leads to a 404 since it redirects to github.com/blob/main/contributing.md and not github.com/CleverCloud/documentation/blob/main/contributing.md

Reply:
I've made some tests and we can just: 
- Use the full URL
- Use a relative URL to github.com

I'll go with the relative one and we'll change later if we find a better solution.